multimodal/src/autogluon/multimodal/data/process_mmlab/process_mmlab_base.py

```diff
@@ -123,13 +121,15 @@ def collate_fn(self, image_column_names: Optional[List] = None, per_gpu_batch_si
         -------
         A dictionary containing one model's collator function for image data.
         """
+        from ...utils import check_if_packages_installed
+
         fn = {}
         if self.requires_column_info:
             assert image_column_names, "Empty image column names."
             for col_name in image_column_names:
                 fn[f"{self.image_column_prefix}_{col_name}"] = StackCollator()
 
-        assert mmcv is not None, "Please install mmcv-full by: mim install mmcv-full."
+        check_if_packages_installed(problem_type=None)
```

Comment:
`problem_type=None` has no effect. I guess the mmcv check is for the collate function. Do we need to check mmcv here?